cli/determined_cli/tensorboard.py

```diff
@@ -37,12 +38,19 @@ def start_tensorboard(args: Namespace) -> None:
         print("Either experiment_ids or trial_ids must be specified.")
         sys.exit(1)
 
+    cxt = None
+    if args.hook is not None:
+        cxt = context.Context.from_local(args.hook, constants.MAX_CONTEXT_SIZE)
+
     config = parse_config(args.config_file, None, [], [])
     req_body = {
         "config": config,
         "trial_ids": args.trial_ids,
         "experiment_ids": args.experiment_ids,
     }
+    if cxt is not None:
+        req_body["hook_data"] = [e.dict() for e in cxt.entries]
```

Comment:
```suggestion
    if args.hook is not None:
        ctx = context.Context.from_local(args.hook)
        req_body["hook_data"] = [e.dict() for e in ctx.entries]
```
and delete the other `if`.